.github/workflows/build_job.yaml

```
@@ -0,0 +1,36 @@
+name: Deploy Operate First GH Pages
+on: 
+     push:
+          branches:
+              - master
+jobs:
+     deploy:
+          runs-on: ubuntu-20.04
+          concurrency:
+              group: ${{ github.workflow }}-${{ github.ref}}
+          steps:
+             - uses: actions/checkout@v2
+             
+             - name: Setup Node
+               uses: actions/setup-node@v2
+               with:
+                   node-version: '14'
+               
+             - name: Dependencies
+               uses: actions/cache@v2
+               with:
+                   path: ~/.npm
+                   key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+                   restore-keys:
+                       ${{ runner.os }}-node-
+
+             - run: npm ci
+             - run: npm build
```

```suggestion
             - run: npm run build
```

`npm build` no longer exist since npm v7, better use `npm run build` (the former `npm build` translates to the the same)

---

Resolved in latest commit.